projects/plugins/jetpack/.gitattributes

```diff
@@ -3,6 +3,7 @@
 /_inc/blocks/**                                          production-include
 /_inc/build/**                                           production-include
 /css/**                                                  production-include
+/extensions/blocks/blocks-manifest.php                   production-include
```

Comment:
Then this can go away (the new file is already covered by line 3)
```suggestion
```

Reply:
Thanks, done in 17ae56bdcfb2